core/metadata-3.3.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IRCv3.3 Metadata
+layout: spec
+copyrights:
+  -
+    name: "Kiyoshi Aman"
+    period: "2016"
+    email: "kiyoshi.aman@gmail.com"
+---
+## Introduction
+
+This specification incorporates the IRCv3.2 metadata specification and
+incorporates it by reference.
+
+## METADATA
+
+### METADATA PRIORITY
+
+This subcommand provides a mechanism for clients to inform the server what
+priorities to assign to particular keys. The default priority is 'on
+subscription' for all keys. The format of `METADATA PRIORITY` MUST be as
+follows:
+
+    METADATA <Scope> PRIORITY <Priority> :<Keys>
+
+`<Scope>` is one of `global`, `channel`, or `user`, reflecting all subscriptions,
+channel-related subscriptions, and subscriptions via `MONITOR`.
+
+`<Priority>` operates in descending order, with `sub` enabling on-subscription
+and on-change notifications, `change` permitting only on-change notifications,
+and `request` requiring that the client explicitly request metadata with the
+relevant keys.
+
+`<Keys>` may optionally be an asterisk (`*`), in which case the priority change
+applies to all keys.
```

Comment:
So long as the client can track what keys it's subscribed to, I don't think this is necessary. If the client is able to understand another key, it can request that key. There's no use to subscribing to keys that the client doesn't understand? Allowing `*` complicates the checking server-side, and without a useful reason I don't think it should be allowed.